docs/orchestrating-elastic-stack-applications/kibana.asciidoc

```diff
@@ -206,6 +206,18 @@ To deploy more than one instance of Kibana, all the instances must share a same
 * `xpack.reporting.encryptionKey`
 * `xpack.encryptedSavedObjects.encryptionKey`
 
+[TIP]
+====
+If you need to access these encryption keys, you can find them using the `kubectl get secrets` command. For example, link:{kibana-ref}/xpack-security-secure-saved-objects.html#encryption-key-rotation[rotating encryption keys] extract the current value of `xpack.encryptedSavedObjects.encryptionKey` in order to set it as an historical encryption value under `xpack.encryptedSavedObjects.keyRotation.decryptionOnlyKeys`.
+
+The secret is named after the corresponding Kibana instance. For example, for a Kibana named `my-kibana`, you can run the following command to retrieve the `xpack.security.encryptionKey` key:
```

Comment:
```suggestion
The secret is named after the corresponding Kibana instance. For example, for a Kibana named `my-kibana`, you can run the following command to retrieve the  current encryption keys:
```

The command you give later on displays all encryption keys not just the `security` one.